src/Components/Endpoints/src/Builder/RazorComponentEndpointConventionBuilder.cs

```diff
@@ -85,7 +85,7 @@ public RazorComponentEndpointConventionBuilder SetServerRenderMode()
     /// Explicitly configures the <see cref="IComponentRenderMode"/> to render exclusively statically.
```

Comment:
This isn't quite what I thought we had discussed. In our IM chat we had `AddWebAssemblyRenderMode` and `AddServerRenderMode` as the method names, and there wasn't an equivalent for SSR.

I understood that nothing like "UseStaticRendering" would be relevant because these modes are purely additive. They don't mean the render mode is **exclusive**. SSR is always available, and people optionally add webassembly and/or server.

Reply:
Hopefully that's only a fairly tiny tweak from what you have here though!

Reply:
Likewise, hopefully `SetRenderModes` is not required and can be removed.

Reply:
I haven't updated this based on our conversations, but I will do so, and put it on a separate commit so that it is clear.

Reply:
I have now updated this to match the new behavior [here](https://github.com/dotnet/aspnetcore/pull/48283/commits/37251fafc78869da83934ac31361b266a2dfb289)